mws/apis/inbound_shipments.py

```diff
@@ -100,7 +99,7 @@ def parse_legacy_item(item: dict, operation: ITEM_OP_TYPE) -> dict:
 
 def parse_shipment_items(
     items: List[Union[InboundShipmentPlanRequestItem, InboundShipmentItem, dict]],
-    operation: Optional[ITEM_OP_TYPE] = None,
+    operation=None,  # type: 'Optional[Literal["CreateInboundShipmentPlan", "CreateInboundShipment", "UpdateInboundShipment"]]'
```

Comment:
This is a hack, for sure, and not ideal (the ideal being we're in 3.8+ land and can use Literal natively).

Those developing on Py<3.8 may get type checker errors if they touch this code, but I expect anyone on those older versions is not developing for the library: they are developing their own code, and the type error is likely hidden from view.

The test suite (going as far back as 3.6) has ensured that this check only occurs when type checkers are active: it does not touch the runtime, so this issue is not visible downstream.